app/controllers/clock_controller.rb

```diff
@@ -47,12 +34,10 @@ def edit
   end
 
   def update
-    current_user.timezone = params[:timezone]
-    if current_user.save
-      redirect_to clock_index_path
-    else 
-      render :new
-    end
+    # current_user.timezone = params[:timezone]
+    current_user.update({timezone: params[:timezone]})
```

Comment:
A couple of things here: Why are you reducing the update action scope to just timezone? Maybe it's a hint that its part of another controller.

Reply:
It is the personal user timezone, and because it is only the name, so I do not need another table to save that value, finally if the value that it is saving for the timezone is empty or nil, the application will take the default timezone